samples/sql-persistence/transitioning-correlation-ids/SqlPersistence_2/Phase1/EndpointOracle/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Persistence.Sql;
+using Oracle.ManagedDataAccess.Client;
+
+partial class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "EndpointOracle";
+
+        var endpointConfiguration = new EndpointConfiguration("EndpointOracle");
+
+        endpointConfiguration.SendFailedMessagesTo("error");
+        endpointConfiguration.UseSerialization<JsonSerializer>();
+        endpointConfiguration.EnableInstallers();
+        var persistence = endpointConfiguration.UsePersistence<SqlPersistence>();
+        var password = Environment.GetEnvironmentVariable("OraclePassword");
+        if (string.IsNullOrWhiteSpace(password))
+        {
+            throw new Exception("Could not extract 'OraclePassword' from Environment variables.");
+        }
+        var username = Environment.GetEnvironmentVariable("OracleUserName");
+        if (string.IsNullOrWhiteSpace(username))
+        {
+            throw new Exception("Could not extract 'OracleUserName' from Environment variables.");
+        }
+        var connection = $"Data Source=localhost;User Id={username}; Password={password}; Enlist=false";
```

Comment:
The SSS adapter calls `Enlist` explicitly if transport uses transaction scope so this should be fine.